src/monarch_ingest/download.yaml

```diff
@@ -258,10 +258,12 @@
   local_name: data/ncbi/gene_info.gz
   tag: ncbi_gene
 
+
+### Now modularized, but here for posterity...
```

Comment:
Better to just remove it, I think

Reply:
Maybe move it all to be comments in the panther ingest repo instead?  (Probably an issue is better for future work, or doc to explain limitations and data not captured, or something?  I think it just doesn't belong here)

Reply:
eh, we can tidy it up later, I'm gonna merge so we can start the kg build